styles.css

```diff
@@ -91,6 +107,172 @@ img {
 }
 
 .social-icon {
-  color: #505f79;
   padding-right: 10px;
+  color: #505f79;
+}
+
+.work {
+  margin: 2rem;
+  display: grid;
+  grid-template-rows: repeat(4, 1fr);
+  grid-row-gap: 2rem;
+}
+
+.single-card {
+  padding: 2rem 1rem;
+  background-color: #fff;
+  border-radius: var(--radius);
+  margin-bottom: 3rem;
+}
+
+.about-portfolio h1 {
+  margin: 1rem 0;
+  font-size: 2rem;
+  font-weight: 800;
+}
+
+.list li:not(.li)::before {
+  content: "\2022";
+  color: #7a869a;
+  font-size: 1rem;
+  font-weight: bolder;
+  display: inline-block;
+  width: 2rem;
+  padding: 0 0.5rem;
+}
+
+.list {
+  margin-right: 1rem;
+  display: flex;
+  justify-content: space-between;
+  font-size: 1rem;
+  font-weight: 600;
+  color: #7a869a;
+}
+
+.li {
+  text-transform: uppercase;
+  color: #344563;
+}
+
+.about-portfolio p {
+  margin: 1rem 0;
+  padding-left: 0;
+  padding-right: 1.5rem;
+  color: #344563;
+}
+
+.tech-used {
+  margin-right: 8rem;
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 2rem;
+}
+
+.tech {
```

Comment:
- [ ] I noticed the tech languages(HTML, CSS, and JavaScript), currently do not add any margin between them. Kindly refer to the template to get the appropriate margin to use.

Reply:
Added margin on the right side.